lib/ElementHandle.js

```diff
@@ -103,6 +103,26 @@ class ElementHandle {
     const objectId = this._remoteObject.objectId;
     return this._client.send('DOM.setFileInputFiles', { objectId, files });
   }
+
+  /**
+   * @return {!Promise<Object>}
+   */
+  async boundingBox() {
+    const box = await this.evaluate(element => {
+      if (!element.ownerDocument.contains(element))
+        return null;
+      const rect = element.getBoundingClientRect();
```

Comment:
Since we are running JavaScript we are are bound to CORS rules, accessing parent frame and getting its position might not be possible (it even happens when ran from the file system).
So the DevTools approach might be better to accomplish this. I'm trying to get this to run:
```js
fit('should handle nested frames', SX(async function() {
      await page.setViewport({width: 500, height: 500});
      await page.goto(PREFIX + '/frames/nested-frames.html');
      const nestedFrame = page.frames()[0].childFrames()[1];
      const elementHandle = await nestedFrame.$('div');
      const box = await elementHandle.boundingBox();
      expect(box).toEqual({ x: 28, y: 253, width: 284, height: 18 });
    }));
```
Currently errors with:
```
  Message:
    Expected $.x = 8 to equal 28.
    Expected $.y = 8 to equal 253.
```